.php-cs-fixer.dist.php

```diff
@@ -178,7 +178,6 @@
                 'switch',
                 'throw',
                 'use',
-                'use_trait',
```

Comment:
Since you do not use traits in this project, the simplest solution is to remove the deprecated `use_trait` option here.